jenkins/jobs/kata-containers-runtime-density-master/config.xml

```diff
@@ -0,0 +1,161 @@
+  </properties>
+  <scm class="hudson.plugins.git.GitSCM" plugin="git@3.8.0">
+    <configVersion>2</configVersion>
+    <userRemoteConfigs>
+      <hudson.plugins.git.UserRemoteConfig>
+        <url>https://github.com/kata-containers/runtime.git</url>
+      </hudson.plugins.git.UserRemoteConfig>
+    </userRemoteConfigs>
+    <branches>
+      <hudson.plugins.git.BranchSpec>
+        <name>refs/heads/master</name>
+      </hudson.plugins.git.BranchSpec>
+    </branches>
+    <doGenerateSubmoduleConfigurations>false</doGenerateSubmoduleConfigurations>
+    <submoduleCfg class="list"/>
+    <extensions/>
+  </scm>
+  <assignedNode>ubuntu_16_04_azure_cloud</assignedNode>
+  <canRoam>false</canRoam>
+  <disabled>false</disabled>
+  <blockBuildWhenDownstreamBuilding>false</blockBuildWhenDownstreamBuilding>
+  <blockBuildWhenUpstreamBuilding>false</blockBuildWhenUpstreamBuilding>
+  <triggers>
+    <com.cloudbees.jenkins.GitHubPushTrigger plugin="github@1.29.0">
+      <spec></spec>
+    </com.cloudbees.jenkins.GitHubPushTrigger>
+  </triggers>
+  <concurrentBuild>true</concurrentBuild>
+  <builders>
+    <hudson.tasks.Shell>
+      <command>#!/bin/bash
+
+set -e
+
+export ghprbPullId
+export ghprbTargetBranch
+
+export METRICS_CI=true
+
+cd $HOME
+git clone https://github.com/kata-containers/tests.git
+cd tests
+
+# Install smem
+sudo apt update
+sudo apt -y install smem
+
+.ci/jenkins_job_build.sh &quot;github.com/kata-containers/runtime&quot;
+
+cd metrics/density 
+sudo -E PATH=$PATH bash docker_memory_usage.sh 1 10 
+sudo -E PATH=$PATH bash memory_usage_inside_container.sh
+
+cd ../results
+memory_usage_file=&quot;memory-footprint.json&quot;
+memory_inside_file=&quot;memory-footprint-inside-container.json&quot;
+
+cat &quot;$memory_usage_file&quot;
+cat &quot;$memory_inside_file&quot;
+units=$(cat &quot;$memory_usage_file&quot; | jq &apos;.Results[0].qemus.Units&apos;)
+units=&quot;${units%\&quot;}&quot; 
+units=&quot;${units#\&quot;}&quot; 
+
+qemu_memory_footprint=&quot;$(cat &quot;$memory_usage_file&quot; | jq &apos;.Results[0].qemus.Result&apos;) $units&quot;
+proxy_memory_footprint=&quot;$(cat &quot;$memory_usage_file&quot; | jq &apos;.Results[0].proxys.Result&apos;) $units&quot;
+shim_memory_footprint=&quot;$(cat &quot;$memory_usage_file&quot; | jq &apos;.Results[0].shims.Result&apos;) $units&quot;
+
+inside_memtotal=&quot;$(cat &quot;$memory_inside_file&quot; | jq &apos;.Results[0].memtotal.Result&apos;) $units&quot;
+inside_memfree=&quot;$(cat &quot;$memory_inside_file&quot; | jq &apos;.Results[0].memfree.Result&apos;) $units&quot;
+
+echo &quot;=================================&quot;
+echo &quot;PSS Measurement:&quot;
+echo &quot;Qemu: $qemu_memory_footprint&quot;
+echo &quot;Proxy: $proxy_memory_footprint&quot;
+echo &quot;Shim: $shim_memory_footprint&quot;
+echo &quot;&quot;
+echo &quot;Memory inside container:&quot;
+echo &quot;Total Memory: $inside_memtotal&quot;
+echo &quot;Free Memory:  $inside_memfree&quot;
```

Comment:
Just a side note - I think here it would be *really* good to show the difference between total and free - as that will be much easier to visually verify when the results are posted to the PR.